src/app/job-board/page.jsx

```diff
@@ -4,35 +4,96 @@ import MdxLayout from '@/components/MdxLayout';
 import { Heading, Span } from '@/components/Typography';
 import { KEY_LINKS } from '@/conference';
 import JobBoardMdx from '@/job-board.mdx';
+import Job from '@/components/Job';
 
-export default function JobBoard() {
+import fetchSheetData from '@/utils/googleSheets';
+
+const fetchJobsData = async () => {
+  try {
+    const sheetId = '1YXfaowCgD8NlOYfQgJukb3yRa9svux6hai9mczdqzbA';
```

Comment:
do update the sheet id once your testing is done

Reply:
sure, let me ask the google sheet admin